packages/examples/src/scenes/logging.tsx

```diff
@@ -0,0 +1,45 @@
+import {Circle} from '@motion-canvas/2d/lib/components';
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {waitFor, waitUntil} from '@motion-canvas/core/lib/flow';
+import {useLogger} from '@motion-canvas/core/lib/utils';
+
+export default makeScene2D(function* (view) {
+  const logger = useLogger();
+
+  view.add(<Circle key="circle" />);
+
+  // Basic Logging
+  yield* waitUntil('basic logging');
+  logger.debug('Just here to debug some code.');
+  logger.info('All fine just a little info.');
+  logger.warn('Be careful somethis has gone wrong.');
+  logger.error('Ups. An error occured.');
```

Comment:
nit: Traditionally this is "Oops", "Woops", or "Whoops" instead of "Ups"

Reply:
Don't know there is a traditional way but love it :D

Reply:
Those are just three options instead of "Ups" for `error`.

Reply:
Yeah, noticed too late as well...

Reply:
No worries 😅